### DISCUSSION TITLE
Request: show both %s and frequencies in chart tooltip

### DISCUSSION THREAD

**Discussion by sfirke** [apache/superset#28216] [Ideas]
**Feature request:** a checkbox in chart design to show percentage along with counts in a tooltip/mouseover.  This would apply to bar charts, area charts, any others where it makes sense.  Currently if a dashboard reader wants to see both percent and counts, they need two redundant charts.

And when Superset displays %s on the y-axis, the tooltip should include the counts.

(From https://apache-superset.slack.com/archives/C0170U650CQ/p1710943570442209 and other sources)

**EDIT: this has been released!**

**Comment by Rusp0** [comment]
Vote for this!
Here, example how it looks like in metabase. 
Would be nice to see the same feature in Superset
![image](https://github.com/apache/superset/assets/80636054/cca517fb-0f2f-47c2-9e6b-b13e84f299dc)

**Reply by clavidk** [reply]
@Rusp0 it seems your request about having totals show is different than what this ticket is about (showing %s along with raw counts)

**Comment by Rusp0** [comment]
> @Rusp0 it seems your request about having totals show is different than what this ticket is about (showing %s along with raw counts)

You think so? I'm sure that's what Sam and I discussed on Slack

**Comment by michael-s-molina** [comment]
https://github.com/apache/superset/pull/27950

**Comment by sfirke** [comment]
This was implemented by the PR that @michael-s-molina linked above and the functionality is in the latest released Superset version (4.1.2) so I'm closing this as completed!
